activerecord/lib/active_record/associations/collection_association.rb

```diff
@@ -26,11 +28,21 @@ module Associations
     class CollectionAssociation < Association #:nodoc:
 
       # Implements the reader method, e.g. foo.items for Foo.has_many :items
-      def reader(force_reload = false)
-        if force_reload
-          klass.uncached { reload }
+      def reader(force_reload = nil, reload: false)
+        unless force_reload.nil?
```

Comment:
Rather than using `nil` here, can we be a little bit more explicit? A user could have theoretically been passing `nil` here, since we were just relying on truthy/falsy.

```
NO_RELOAD_ARGUMENT = Object.new

def reader(force_reload = NO_RELOAD_ARGUMENT, reload: false)
  unless force_reload == NO_RELOAD_ARGUMENT
```